package.json

```diff
@@ -16,6 +16,26 @@
   "author": "brian@hovercraftstudios.com",
   "license": "MIT",
   "devDependencies": {
-    "lerna": "^2.1.2"
+    "@briancavalier/assert": "^3.2.0",
+    "babel-core": "^6.26.0",
+    "babel-plugin-annotate-pure-calls": "^0.2.0",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-transform-builtin-extend": "^1.1.2",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-power-assert": "^1.0.0",
```

Comment:
couldnt find if it was used before or if this is some leftover from the past, ive kept it for now but it seems this can be either removed or added to the used presets in `prelude` package

Reply:
Oops.  Yes, we can get rid of it.